_returners/splunk_nebula_return.py

```diff
@@ -94,6 +94,8 @@ def returner(ret):
     jid = ret['jid']
     master = __grains__['master']
     fqdn = __grains__['fqdn']
+    # Sometimes fqdn is blank. If it is, replace it with minion_id
+    fqdn = minion_id if fqdn == "" else fqdn
```

Comment:
Can we change `if fqdn == ''` to `if not fqdn`? That will handle empty string, but also None or other falsey values. Same with the other two files.